src/ORM/DatabaseAdmin.php

```diff
@@ -38,6 +41,7 @@ class DatabaseAdmin extends Controller
 
     /**
      * Obsolete classname values that should be remapped in dev/build
+     * @deprecated 5.4.0 Will be replaced with SilverStripe\Dev\Command\DbBuild.classname_value_remapping
```

Comment:
```suggestion
```
Class is already deprecated and property is the same name

Reply:
Discussion can be in https://github.com/silverstripe/silverstripe-framework/pull/11365#discussion_r1755986594